Sources/Verge/Store/StoreType+SwiftUI.swift

```diff
@@ -41,4 +41,35 @@ extension BindingDerived {
   }
 }
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+extension StoreType {
+  /// Generates a SwiftUI.Binding that gets and updates the StoreType.State.
+  /// Usage:
+  ///
+  ///     TextField("hoge", text: store.binding(\.inputingText))
+  ///
+  /// - Warning: Still in experimentals.
+  /// - Parameters:
+  ///   - keypath: A property of the state to be bound.
+  ///   - mutation: A closure to update the state.
+  ///   If the closure is nil, state will be automatically updated.
+  /// - Returns: The result of binding
+  public func binding<T>(_ keypath: WritableKeyPath<State, T>, with mutation: ((T) -> Void)? = nil) -> Binding<T> {
```

Comment:
It would be better to add the same method in StoreComponentType which just calls directly StoreType's one.

Reply:
or if you don't use StoreComponentType, we don't need to do this.

Reply:
✅  https://github.com/VergeGroup/Verge/pull/256/commits/6a55ea3f96b213952e2c765aab9cc5a1988e7d80